IronRoadForChildren/Features/Program/Day/ProgramItemView.swift

```diff
@@ -8,27 +8,35 @@
 import SwiftUI
 
 struct ProgramItemView: View {
+	let event: Event
+
 	private let rowHeight: CGFloat = 100
 
 	var body: some View {
 		HStack(spacing: 10) {
-			Image("seiler-speer")
-				.resizable()
-				.scaledToFill()
-				.frame(width: rowHeight, height: rowHeight)
-				.clipped()
+			if let url = URL(string: event.picture.path) {
+				AsyncImage(url: url) { image in
```

Comment:
I guess AsyncImage automatically handles invalid urls (e.g. 404, or not an image), right?